code/mod_table/mod_table.cpp

```diff
@@ -92,6 +92,7 @@ bool Show_subtitle_uses_pixels;
 int Show_subtitle_screen_base_res[2];
 int Show_subtitle_screen_adjusted_res[2];
 bool Always_warn_player_about_unbound_keys;
+bool Always_use_distant_firepoints;
```

Comment:
You need an additional entry resetting this value to false in ``mod_table_reset()``, since otherwise it'd be totally unset when the table option isn't specified. This _should_ be false for booleans, but it's not proper style to rely on that.